client/src/standalone/videos/embed.ts

```diff
@@ -17,6 +18,8 @@ import { PeerTubeEmbedApi } from './embed-api'
 import { TranslationsManager } from '../../assets/player/translations-manager'
 import videojs from 'video.js'
 import { VideoJSCaption } from '../../assets/player/peertube-videojs-typings'
+import { AuthUser } from '../../app/core/auth/auth-user.model'
```

Comment:
Don't import things from angular app

Reply:
Create a `root-helpers` directory outside (or use the `shared` root directory) `app` and put things you need in there.

You'll have to split `AuthUser` because we don't really want to import rxjs (create a parent class or a mixin)